specs/src/specs/block_manager.md

```diff
@@ -0,0 +1 @@
+../../../block/block.md
```

Comment:
This should not be changed since it breaks: https://github.com/rollkit/rollkit/actions/runs/6579744136/job/17876271122?pr=1205